rules/falco-incubating_rules.yaml

```diff
@@ -16,40 +16,61 @@
 # limitations under the License.
 #
 
-# Information about rules tags and fields can be found here: https://falco.org/docs/rules/#tags-for-current-falco-ruleset
-# The initial item in the `tags` fields reflects the maturity level of the rules introduced upon the proposal https://github.com/falcosecurity/rules/blob/main/proposals/20230605-rules-adoption-management-maturity-framework.md
-# `tags` fields also include information about the type of workload inspection (host and/or container), and Mitre Attack killchain phases and Mitre TTP code(s)
+# Information about rules tags and fields can be found here:
+# https://falco.org/docs/rules/#tags-for-current-falco-ruleset
+# The initial item in the `tags` fields reflects the maturitylevel of the rules
+# introduced upon the proposal
+# https://github.com/falcosecurity/rules/blob/main/proposals/
+# 20230605-rules-adoption-management-maturity-framework.md
+# `tags` fields also include information about the type of
+# workload inspection (host and/or container), and Mitre Attack killchain phases
+# and Mitre TTP code(s)
 # Mitre Attack References:
 # [1] https://attack.mitre.org/tactics/enterprise/
-# [2] https://raw.githubusercontent.com/mitre/cti/master/enterprise-attack/enterprise-attack.json
+# [2] https://raw.githubusercontent.com/mitre/cti/master/enterprise-attack/
+#     enterprise-attack.json
 
 # Starting with version 8, the Falco engine supports exceptions.
 # However the Falco rules file does not use them by default.
+---
+
 - required_engine_version: 0.31.0
 
 - macro: open_write
-  condition: (evt.type in (open,openat,openat2) and evt.is_open_write=true and fd.typechar='f' and fd.num>=0)
+  condition: (evt.type in (open,openat,openat2) and evt.is_open_write=true and
+     fd.typechar='f' and fd.num>=0)
 
 - macro: open_read
-  condition: (evt.type in (open,openat,openat2) and evt.is_open_read=true and fd.typechar='f' and fd.num>=0)
+  condition: >
+    (evt.type in
+      (
```

Comment:
Personally not a fan of this formatting here and it's inconsistent with above's `open_write`.

Reply:
@incertum i changed this formatting.